network/electrumx-server.rst

```diff
@@ -0,0 +1,241 @@
+not present, add them as shown below.
+
+::
+
+  rpcuser=XXXXXXXXXXXXX
+  rpcpassword=XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  rpcallowip=127.0.0.1
+
+Replace the fields marked with ``XXXXXXX`` as follows:
+
+- ``rpcuser``: enter any string of numbers or letters, no special
+  characters allowed
+- ``rpcpassword``: enter any string of numbers or letters, no special
+  characters allowed
+
+
+Transaction Indexing
+--------------------
+
+Make sure the following line is present in ``dash.conf``. If it's not present,
+add it and restart the node using the ``-reindex`` option. Note that reindexing
+can take a long time.
+
+.. code-block:: shell
+  
+  txindex=1
+
+Install Dependencies
+====================
+
+Python 3.7 is required for ElectrumX, ``build-essential`` and ``-dev`` version
+of Python are required for ``x11_hash``, and ``daemontools`` is required to
+manage the ElectrumX service.
+
+Install Python 3.7 and the required build tools via the package system:
+
+.. code-block:: shell
+
+  sudo add-apt-repository ppa:deadsnakes/ppa # the repo with python3.7
+  sudo apt update
+  sudo apt install -y software-properties-common build-essential daemontools python3.7-dev python3-pip
+
+Now that Python 3.7 is installed, make it the default version:
+
+.. code-block:: shell
+
+  sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1
+
+Since the default version of Python has potentially changed, reinstall
+setuptools and then install X11 library:
+
+.. code-block:: shell
+
+  pip3 install --upgrade --force-reinstall setuptools
+  pip3 install x11_hash
+
+Generate an SSL key and an associated certificate file for the server:
+
+.. code-block:: shell
+
+  openssl genrsa -out server.key 2048
+  openssl req -new -key server.key -out server.csr
+  openssl x509 -req -days 1825 -in server.csr -signkey server.key -out server.crt
+
+Install and Configure ElectrumX
+===============================
+
+Install ElectrumX
+-----------------
+
+Clone ElectrumX from GitHub via git:
+
+.. code-block:: shell
+
+  git clone https://github.com/spesmilo/electrumx.git
+
+
+Build and install the project:
+
+.. code-block:: shell
+
+  cd electrumx && sudo python3 setup.py install && pip3 install . --upgrade && cd
+
+
+Configure ElectrumX
+-------------------
+
+Create the ElectrumX directories and copy files into them. These directories will be
+located in the ``electrumx`` user's home directory:
+
+.. code-block:: shell
+
+  mkdir -p scripts service data/electrumx-server var/log/electrumx
+  cp -R /home/electrumx/electrumx/contrib/daemontools scripts/electrumx
+ 
+Set the server banner message and the coin type that the server will host:
+
+.. code-block:: shell
+
+  echo '$SERVER_SUBVERSION running on $DAEMON_SUBVERSION' > /home/electrumx/electrumx/banner
+  echo "/home/electrumx/electrumx/banner" > scripts/electrumx/env/BANNER_FILE
+  echo "Dash" > scripts/electrumx/env/COIN
+
+Set network values
```

Comment:
Is this a heading? Should be underlined.

Reply:
I couldn't decide and forgot to do either. Both seem a little weird, but I'll add a colon to be consistent with the preceding ones